batch/test/test_batch.py

```diff
@@ -598,6 +598,16 @@ def test_verify_no_access_to_metadata_server(client):
     assert "Connection timed out" in j.log()['main'], (j.log()['main'], status)
 
 
+def test_can_use_hailctl_auth(client):
+    builder = client.create_batch()
+    j = builder.create_job(os.environ['CI_UTILS_IMAGE'],
+                           'gcloud', 'auth', 'list')
+    status = j.wait()
+    assert status['state'] == 'Success', f'{j.log(), status}'
+    log = j.log()
+    assert '*       test-665@hail-vdc.iam.gserviceaccount.com' in log, f'{j.log(), status}'
```

Comment:
Will this be true for tests running in the default namespace? I can't remember how many service accounts we have going around.

Reply:
The test-gsa-key is one of the secrets copied from default by the `createNamespace` that creates `default`, so it should all be the same account.
```
(base) # k get secret test-gsa-key -o json  | jq '.data["key.json"]' -r | base64 -D | jq '.client_email'
"test-665@hail-vdc.iam.gserviceaccount.com"

```